src/server_manager/messages/sv.json

```diff
@@ -240,8 +242,8 @@
   "setup-cancel": "Avbryt när du vill",
   "setup-create": "Skapa server",
   "setup-description": "Har du ingen server? Skapa ett konto med DigitalOcean.",
-  "setup-do-cost": "Endast 50 kronor i månaden",
```

Comment:
This is wrong. It says 50 Swedish kronor. We can't convert that.

Reply:
This is the old translation, it's updated to 60 SEK, is that correct?